From 4ec9b56716af33331198c963e23c3e0e8ee75c87 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Fri, 8 Feb 2019 17:05:03 +0200
Subject: [PATCH 5/6] legacy: Ignore neverallows

* We currently have about 1800+ neverallows,
  let the build proceed until the day we clean it up

Change-Id: I35a39ec73c0047f93124f72249a6fa936fe0875a
---
 sepolicy.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sepolicy.mk b/sepolicy.mk
index 4511aa0..9edc56e 100644
--- a/sepolicy.mk
+++ b/sepolicy.mk
@@ -7,6 +7,8 @@ BOARD_SEPOLICY_DIRS += \
 ifneq (,$(filter userdebug eng, $(TARGET_BUILD_VARIANT)))
 BOARD_SEPOLICY_DIRS += \
     device/qcom/sepolicy-legacy/test
+
+SELINUX_IGNORE_NEVERALLOWS := true
 endif
 
 BOARD_PLAT_PUBLIC_SEPOLICY_DIR += \
-- 
2.17.1

